great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -509,6 +513,26 @@ def get_domain_records(
         if TextClause and isinstance(selectable, TextClause):
             selectable = selectable.columns().subquery()
 
+        filter_conditions: List[RowCondition] = domain_kwargs.get(
+            "filter_conditions", []
+        )
+        # For SqlAlchemyExecutionEngine only one filter condition is allowed
+        if len(filter_conditions) == 1:
+            filter_condition = filter_conditions[0]
```

Comment:
@anthonyburdi So `filter_conditions` will now absorb `row_condition`?  If so, what about backward compatibility?  Thanks.

Reply:
Row conditions are still processed, and I moved them now to above the `filter_conditions` as discussed.